src/Http/Cookie/CookieInterface.php

```diff
@@ -49,6 +49,15 @@ public function getName();
     public function getValue();
 
     /**
+     * Gets the cookie value as a string.
+     *
+     * This will collapse any complex data in the cookie with json_encode()
+     *
+     * @return string
+     */
+    public function getStringValue();
```

Comment:
Any reason not to use `__toString`?

Reply:
Cookies have more properties than just the value. Wouldn't the string value of a cookie include the Secure, HttpOnly, Path and Domain bits?

Reply:
Yep, you're absolutely right! Disregard :)